.github/actions/publish-libs/action.yml

```diff
@@ -0,0 +1,115 @@
+name: "Publish ACA libs to NPM and GH registry"
+description: "Publish ACA libs to NPM and GH registry"
+
+inputs:
+  github_token:
+    description: 'Github token'
+    required: true
+    type: string
+  npm-registry-address:
+    description: 'NPM registry address'
+    required: true
+    type: string
+  npm-registry-token:
+    description: 'NPM registry token'
+    required: true
+    type: string
+  dry-run:
+    description: dry run flag
+    required: true
+    type: boolean
+
+runs:
+  using: "composite"
+  steps:
+    - name: update libs version
+      shell: bash
+      run: |
+        DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+        ROOT_DIR=${DIR}/../../..
```

Comment:
probably not needed

when you really need an absolute path to the directory where the repo is checkout, use $GITHUB_WORKSPACE

Reply:
Removed